client/src/main/java/io/split/engine/matchers/EqualToMatcherSemver.java

```diff
@@ -0,0 +1,54 @@
+package io.split.engine.matchers;
+
+import io.split.engine.evaluator.EvaluationContext;
+
+import java.util.Map;
+
+public class EqualToMatcherSemver implements Matcher {
```

Comment:
Can we rename the class to EqualToSemverMatcher? To be consistent with the spec and the other SDKs.